### PR TITLE
added more projects and organizations

### PR DESCRIPTION
**Pull request by omazapa**
More updates from different projects and organizations.

Best,
Omar.

### CODE REVIEW

**Comment by jouvin** [comment]
@omazapa please could you use the syntax used for other projects to add images (using the Markdown syntax)?

**Comment by omazapa** [comment]
@jouvin sure, the code was fixed.

File: _gsocproposals/proposal_Geant4PRNG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    * When a secondary is created, attach to it either a seed or a new state for PRNG, for use in that track.
+    * Participating in extending the interface different PRNGs to provide different ways to choose the state of PRNG for a secondary.
+    * Create implementations including MCRGs, counter-based PRNGs and novel PRNGs with guarantees of stream independence.
+ 
+
+**Requirements**: programming in C/C++ or Java at least for class projects is required. Use of Linux/Unix, cmake and/or a coding IDE (Eclipse, Xcode) will be assets.
+Knowledge gained: use of Pseudo-Random Number Generators and state of the art of PRNGs.
+
+**Mentor**: 
+* John Apostolakis​ sft-gsoc@cern.ch
```
Comment (jouvin) [inline _gsocproposals/proposal_Geant4PRNG.md]: Any good reason not to use the usual syntax `[name](mailto:email)`?

File: _gsocproposals/proposal_Geant4RK.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * Introduce new methods for the control of integration error of existing integration Runge Kutta methods.
+ * Create heuristics for choosing an appropriate method (helix, RK of particular order, Burlich/Stoer), using knowledge of the accuracy requirements and the variation of the derivative in the proposed step.
+ 
+**Expected results**: Working implementation of
+* improved alternative error control methods for RK integration, and/or 
+* integration methods which combine different methods, e.,g. RK methods of different order for improved performance.
+
+**Requirements**: This project requires prior exposure to Numerical Analysis and familiarity with either C++, C or Java programming.  Exposure to either numerical methods for solving Ordinary Differential equations (ODEs) or tools for analysing data such as ROOT or R will be valuable. Both programming skill and knowledge of numerical methods for ODEs will be improved by undertaking this project.
+
+**Mentor**: John Apostolakis​ sft-gsoc@cern.ch
```
Comment (jouvin) [inline _gsocproposals/proposal_Geant4RK.md]: Idem... and probably in all proposals...

**Comment by jouvin** [comment]
@omazapa sorry I had not time this morning to look at all the files... Still one detail with email address (look at the comments) and as for me this is ready for merging.

**Comment by hegner** [comment]
I did the fixes for you now as not to suffer from the timezone difference.

**Comment by omazapa** [comment]
@jouvin and @hegner  yes thank you guys, I am in very different timezone now.
I will be more careful with those details in the next pull request.
Best Regards
Omar.

**Comment by jouvin** [comment]
@omazapa no problem, thanks for your contributions! In fact we are working on our site (quite some progress has been done in the last days) to make all of this easier for people who wants to add information, exposing less of the Markdown and Jekyll gory details!

**Pull request merged**